pspgl/PSPBUILD

```diff
@@ -0,0 +1,50 @@
+# Maintainer: Wouter Wijsman <wwijsman@live.nl>
+pkgname=pspgl
+pkgver=r59.2e82204
+pkgrel=1
+pkgdesc="OpenGL-ESish library for PSP"
+arch=('mips')
+url="https://github.com/pspdev/psp-ports"
+license=('BSD')
+depends=()
+makedepends=()
+optdepends=()
+source=("git+https://github.com/pspdev/psp-ports.git")
+sha256sums=('SKIP')
+
+pkgver() {
+  cd psp-ports
+  # Set the revisioin of the repo as version
+  # Means any update to psp-ports will result in a new version number
```

Comment:
This versioning will not work out. As said we want to move all psp-ports out in separate repos.

Reply:
Yeah, agreed. This was probably the best solution for when using the psp-ports repo, but it's not a good one.

Reply:
It now uses the revision of the specific directory within psp-ports instead. So if another package gets upgraded, it does not affect the version of pspgl.